server/basicServices/basicServices/GenericRepresentation.js

```diff
@@ -117,6 +117,10 @@ exports.getResponseValueList = function (operationName) {
             let responseInstanceFieldName;
             let responseInstanceValue;
             let responseInstancedataTypeOfValue;
+            let fieldNameDataUpdatePeriod = "dataUpdatePeriod"
+            let dataUpdatePeriodEnum = {
+                "real-time": "http-server-interface-1-0:DATA_UPDATE_PERIOD_TYPE_REAL_TIME"
```

Comment:
As per specification dataUpdatePeriodEnum contains 4 types but here only one is declared can we add the other 3 also

Reply:
done